requirements-dev.txt

```diff
@@ -1,10 +1,8 @@
 -e .[all,server]
 -r requirements-tests.txt
 
-black==22.1.0
+black==22.3.0
 flake8==4.0.1
-# https://github.com/pallets/click/issues/2232
-click<8.1.0
```

Comment:
We do need to keep `click` as a requirement though :)

Reply:
indeed `click` is required by scripts/scaffold.py